src/mozilla_sec_eia/models/sec_eia_record_linkage/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""Implement record linkage model between SEC companies and EIA utilities."""
+
+from dagster import Definitions, load_assets_from_modules
+from dagstermill import (
+    ConfigurableLocalOutputNotebookIOManager,
+)
+from upath import UPath
+
+from mozilla_sec_eia.library import model_jobs
+from mozilla_sec_eia.library.generic_io_managers import (
+    PandasParquetIOManager,
+    PickleUPathIOManager,
+)
+from mozilla_sec_eia.library.mlflow import (
+    MlflowPyfuncModelIOManager,
+    mlflow_interface_resource,
+    mlflow_train_test_io_managers,
+)
+from mozilla_sec_eia.models.sec10k.utils.cloud import cloud_interface_resource
+
+from . import transform_eia_input, transform_sec_input
+
+eia_assets = load_assets_from_modules([transform_eia_input])
+sec_assets = load_assets_from_modules([transform_sec_input])
+
+eia_input_table_production_job = model_jobs.create_production_model_job(
+    "eia_input_table_creation", transform_eia_input.production_assets
+)
+sec_input_table_production_job = model_jobs.create_production_model_job(
+    "sec_input_table_creation", transform_sec_input.production_assets
+)
+
+defs = Definitions(
+    sec_assets,
+    jobs=[eia_input_table_production_job, sec_input_table_production_job],
```

Comment:
Hey @zschira , I'm trying to test the creation of these assets but I don't see my job showing up in the dagster jobs.  Do i need to add the job somewhere else as well?